hopp_tools_steel.py

```diff
@@ -555,39 +562,58 @@ def run_HOPP(
         wind_size_mw = forced_wind_size
         storage_size_mw = forced_storage_size_mw
         storage_size_mwh = forced_storage_size_mwh
-        storage_hours = 0
+        if storage_size_mw>0:
+            storage_hours = storage_size_mwh/storage_size_mw
+        else:
+            storage_hours = 0
+        
 
     turbine_rating_mw = scenario['Turbine Rating']
     tower_height = scenario['Tower Height']
     rotor_diameter = scenario['Rotor Diameter']
     
     if floris == False:
-        if storage_size_mw > 0:
-            technologies = {#'pv':
-                            #   {'system_capacity_kw': solar_size_mw * 1000},
-                            'wind':
-                                {'num_turbines': np.floor(wind_size_mw / turbine_rating_mw),
+        technologies={}
+        if run_wind_plant:
```

Comment:
Instead of being a flag, `run_wind_plant` checks should be replaced with a check on the wind size as is done with the `solar_size_mw` to keep things consistent and allow similar usage.

Reply:
Is this change recommended for this function only or all cases where we use run_wind_plant? If it's just in this function, then that can be done easily. If the change is suggested for more than that, it gets more complicated.

If just in this function: This would require setting wind_size_mw=0 in the main run script (or making a new variable that we set to the actual installed wind capacity when running the wind plant and setting to zero when not running the wind plant). If we set wind_size_mw=0 in the run file, although we are using wind power just not running the wind farm model, then it could have some downstream impacts and cause issues (notably in the lcoh calculation). My idea would be to go the route of using a new variable, which would be set depending on the run_wind_plant flag in the run script.

To remove the run_wind_plant flag in the main run script and keep the same parametric sweep functionality, then I think we'd need to introduce a variable called wind_sizes_mw which is a list of length = len(solar_sizes_mw)*len(storage_sizes_mw), the first value is the actual wind capacity, all the rest are zero. That way, wind_size_mw is always the installed wind capacity (hopefully preventing any potential downstream issues), but the run_wind_plant flag is replaced with "if wind_sizes_mw[j]>0" and pass in wind_sizes_mw[j] to the run_HOPP function (that way it works with the change I suggested previously).